src/qobuz/qobuzrequest.cpp

```diff
@@ -1079,12 +1079,17 @@ void QobuzRequest::ParseSong(Song &song, const QJsonObject &json_obj, const Arti
 
   QString title = json_obj["title"].toString();
   int track = json_obj["track_number"].toInt();
+  int disc = 1;
```

Comment:
This should be `0` if Qobuz does not return disc, so the collection model does not show disc in the "Album - Disc" grouping, unless it's an album with multiple discs.